<?xml version="1.0"?>
<robot name="ZumoCarto">
    <link name="map"/>
    <link name="imu"/>
    <link name="odom"/>
    <link name="base">
        <visual>
            <origin rpy="0 0 0" xyz="0 0 0"/>
            <geometry>
                <mesh filename="package://ZumoCarto/urdf/zumo.STL" scale="0.1 0.1 0.1"/>
            </geometry>
            <material name="Grey">
                <color rgba="0.30 0.30 0.10 1.0"/>
            </material>
        </visual>
    </link>
    
    <joint name="joint1" type="planar">
        <parent link="map"/>
        <child link="base"/>
        <axis xyz="0 0 1"/>
        <origin rpy="0 0 0" xyz="0 0 0"/>
        <!--<limit effort="1000" lower="-10" upper="10" velocity="0.2"/>-->
        </joint>
    <joint name="jointimu" type="fixed">
        <parent link="base"/>
        <child link="imu"/>
        <origin rpy="0 0 0" xyz="0 0 0"/>
    </joint>
    <joint name="jointodom" type="fixed">
        <parent link="base"/>
        <child link="odom"/>
        <origin rpy="0 0 0" xyz="0 0 0"/>
    </joint>
  
  
</robot>